extendeddatacrossword.go

```diff
@@ -14,24 +14,29 @@ const (
 // ErrUnrepairableDataSquare is thrown when there is insufficient chunks to repair the square.
 var ErrUnrepairableDataSquare = errors.New("failed to solve data square")
 
-// ErrByzantineRow is thrown when a repaired row does not match the expected row Merkle root.
-type ErrByzantineRow struct {
-	RowNumber uint     // Row index
-	Shares    [][]byte // Pre-repaired row shares. Missing shares are nil.
+// ErrByzantineRowOrCol is thrown when a repaired row or column does not match the expected row or column Merkle root.
+type ErrByzantineRowOrCol struct {
+	isRow  bool     // Is row.
```

Comment:
Can we make a separate enum type for it? Something like: 
```go
type Axis int // or some other better generic name

const (
	Row Axis = iota
	Col
)

```

Reply:
Fun fact there's already an enum for that but it's unused :joy:. Probably from some older code that used it to branch.

Reply:
Fixed in [780a5bf](https://github.com/celestiaorg/rsmt2d/pull/80/commits/780a5bf8683982e369168079c758da86dbd072af)